x-pack/test/security_solution_api_integration/test_suites/investigation/saved_objects/trial_license_complete_tier/helpers.ts

```diff
@@ -53,82 +53,26 @@ export const deleteAllNotes = async (es: Client): Promise<void> => {
   });
 };
 
-export const createNoteAssociatedWithDocumentOnly = async (
+export const createNote = async (
   supertest: SuperTest.Agent,
-  documentId: string,
-  noteText: string
+  note: {
+    documentId?: string;
+    savedObjectId?: string;
+    user?: string;
+    text: string;
+  }
 ) =>
   await supertest
     .patch(NOTE_URL)
     .set('kbn-xsrf', 'true')
     .send({
       note: {
-        eventId: documentId,
-        timelineId: '',
+        eventId: note.documentId || '',
+        timelineId: note.savedObjectId || '',
         created: Date.now(),
-        createdBy: 'elastic',
+        createdBy: note.user || 'elastic',
         updated: Date.now(),
-        updatedBy: 'elastic',
-        note: noteText,
-      },
-    });
-
-export const createNoteAssociatedWithSavedObjectOnly = async (
-  supertest: SuperTest.Agent,
-  savedObjectId: string,
-  noteText: string
-) =>
-  await supertest
-    .patch(NOTE_URL)
-    .set('kbn-xsrf', 'true')
-    .send({
-      note: {
-        eventId: '',
-        timelineId: savedObjectId,
-        created: Date.now(),
-        createdBy: 'elastic',
-        updated: Date.now(),
-        updatedBy: 'elastic',
-        note: noteText,
-      },
-    });
-
-export const createNoteAssociatedWithDocumentAndSavedObject = async (
-  supertest: SuperTest.Agent,
-  documentId: string,
-  savedObjectId: string,
-  noteText: string
-) =>
-  await supertest
-    .patch(NOTE_URL)
-    .set('kbn-xsrf', 'true')
-    .send({
-      note: {
-        eventId: documentId,
-        timelineId: savedObjectId,
-        created: Date.now(),
-        createdBy: 'elastic',
-        updated: Date.now(),
-        updatedBy: 'elastic',
-        note: noteText,
-      },
-    });
-
-export const createNoteAssociatedWithNothing = async (
-  supertest: SuperTest.Agent,
-  noteText: string
-) =>
-  await supertest
-    .patch(NOTE_URL)
-    .set('kbn-xsrf', 'true')
-    .send({
-      note: {
-        eventId: '',
-        timelineId: '',
-        created: Date.now(),
-        createdBy: 'elastic',
-        updated: Date.now(),
-        updatedBy: 'elastic',
-        note: noteText,
-      },
+        updatedBy: note.user || 'elastic',
+        note: note.text,
+      } as BareNote,
```

Comment:
This should probably be `PersistNoteRouteRequestBody`

Reply:
that makes sense, if that's ok I made the change in this other PR ([here](https://github.com/elastic/kibana/pull/195501)) that way it saves some CI time... ?